Filter user defined channels by size (#13861)
friendica/friendica: doc/Channels.md

Additional keywords for the full text search
---

> Additionally to the search for content, there are additional keywords that can be used in the full text search. Alternatives are presented via "|".

Reviewer: ```suggestion
Additionally to the search for content, there are keywords that can be used in the full text search.
Alternatives are presented with "|".
```

Author: Changes are done and also the merge commit is solved.
